doc/use.md

```diff
@@ -0,0 +1,37 @@
+# How to use Learn-OCaml autogen?
+
+Suppose we have a Learn-OCaml exercises directory. We want to create a new
```

Comment:
Give a name (or a symbolic name) to this directory so that you can refer to it in subsequent commands and text.